src/main/java/minicraft/core/io/Localization.java

```diff
@@ -110,12 +144,23 @@ public static void loadLanguage() {
 				localization.put(key, json.getString(key));
 			}
 		}
+
+		for (String text : unloadedLocalization.get(DEFAULT_LOCALE)) {
+			json = new JSONObject(text); // This JSON has been verified before.
+			// Put all loc strings in a key-value set.
+			for (String key : json.keySet()) {
+				fallbackLocalization.put(key, json.getString(key));
+			}
```

Comment:
These comments doesn't say anything more to the user other than what they can immediately see.

Reply:
What?

Reply:
The comment's aren't very informative.

Reply:
There's still not a comment that tells why this for loop exists.

Reply:
I have added.

Reply:
It's kinda hidden in plain sight. Ideally code shouldn't need documentation, but in reality almost all code needs documentation. So, you need documentation, but since there are so many other lines that tell us absolutely no useful information, the actually useful comments are harder to find.

Be more picky about what you're actually documenting. Stop adding useless comments like: `Put all loc strings in a key-value set.` `Debugging locale.` `This JSON has been verified before.` (what does this even mean?) `Incrementing index.` `Decrementing index.`

Reply:
That for loop snap is outdated. So, the comments you mentioned should be removed?

Reply:
Yes, but use your own common sense and decide what is relevant or not.